### PR TITLE
WAF samples - health alerts sample update

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

File: Reliability/HealthAlerts/Readme.md [hunk 1]
```diff
@@ -33,34 +32,62 @@ like so:
...
 
-You will be prompted for the ActivityLogAlertName, enter any AlertName you want.
+Create a new resource group to deploy the sample
+
+```powershell
+New-AzResourceGroup -Name HealthAlerts-RG -Location "Central US"
+```powershell
 
-You will also be prompted for the ActionGroupResourceId, which is composed this way:
+To start a new deployment using the template provided in this sample, use the powershell command below; you will also be prompted for the ActionGroupResourceId, which is composed this way (replace theplace holder with your subscriptionId):
```
Comment (ckittel) [inline Reliability/HealthAlerts/Readme.md:58]:
```suggestion
To start a new deployment using the template provided in this sample, use the PowerShell command below; you will also be prompted for the ActionGroupResourceId, which is composed this way (replace the _subscriptionId_ placeholder with your subscription ID):
```

File: Reliability/HealthAlerts/Readme.md [hunk 2]
```diff
@@ -33,34 +32,62 @@ like so:
...
+Timestamp               : 11/28/2021 10:00:04 PM
+Mode                    : Incremental
+TemplateLink            :
+Parameters              :
+                          Name                     Type                       Value
+                          =======================  =========================  ==========
+                          activityLogAlertName     String                     activityLog-alert-1
+                          actionGroupResourceId    String                     /subscriptions/a012a8b0-522a-4f59-81b6-aa
+                          0361eb9387/resourceGroups/HealthAlerts-RG/providers/microsoft.insights/actionGroups/health-al
+                          erts-ag
```
Comment (ckittel) [inline Reliability/HealthAlerts/Readme.md:89]:
```suggestion
                          actionGroupResourceId    String                     /subscriptions/subscriptionId/resourceGroups/HealthAlerts-RG/providers/microsoft.insights/actionGroups/health-alerts-ag
```

**Review by ckittel — APPROVED** [review]
Only a couple minor suggestions below, but nothing blocking.

**Comment by magrande** [comment]
Thanks @ckittel  !!

**Pull request merged**